Sources/CodeGen/CXX/CXXTranspiler.swift

```diff
@@ -169,597 +128,470 @@ public struct CXXTranspiler {
         unreachable("unexpected class member")
       }
     }
-
-    // Create the C++ class.
-    module.addTopLevelDecl(CXXClassDecl(name: name, members: cxxMembers, original: decl))
+    return cxxMembers
+  }
+
+  /// Translate a binding decl contained in a product type into one or more C++ class attributes.
+  /// One class attribute will be generated for each name in the binding decl.
+  /// The resulting class members are appended to `res` (the array of class members).
+  private func cxx(
+    productTypeBinding src: BindingDecl.Typed, into res: inout [CXXClassDecl.ClassMember]
+  ) {
```

Comment:
- [Don't use abbrevs](https://www.swift.org/documentation/api-design-guidelines/#avoid-abbreviations). There's nothing wrong with a single-character name if alternatives are too long. Otherwise, `source` and `target` would be suitable.  
- Don't write documentation words that simply restate the types in the signature (“the array of class members”)

```suggestion
  /// Appends one C++ class attribute to `target` for each name bound in `source`.
  private func transpile(
    productTypeBinding source: BindingDecl.Typed, into target: inout [CXXClassDecl.ClassMember]
  ) {
```

I hope this gives some idea of what we're going for.

Reply:
done